graphql/api.js

```diff
@@ -256,6 +257,38 @@ export const updateApplication = async (id, company, role, currentExcitement) =>
 
   const data = await res.json();
 
+  return {
+    data: getData(data),
+    errorMessage: getErrorMessage(null, data)
+  };
+};
+
+export const updateInterviewNotes = async (id, notes, date, type) => {
+  const query = `mutation updateInterviewNotes($id: ID!, $notes: String!, $date: String!, $type: String!) {
+    updateInterview(id: $id, data:{
+      date: $date
+      type: $type
+      notes: $notes
+      }){
+    			notes
```

Comment:
ah i see what you mean about the indentation. I wonder if its a tabs vs spaces issue.

Reply:
same, the linter will hopefully fix this